pkg/api/login_handler.go

```diff
@@ -21,11 +21,16 @@ const (
 	JWTCookieName          = "access_token"
 )
 
-func GenerateJWT(secret []byte, userID int, issuedAt, expiresAt time.Time) (string, error) {
+// Be aware! The aud field should *only* be used for new applications! Otherwise it should be passed as an empty string, so it supports
+// backward compatibility. As well Id field contains the email and not the Id for backward compatibility. The secret field is the secret key
```

Comment:
This will be a security and functionality headache: whenever a user is used, there is no way to know whether it's a new user-with-email or an old user-without-email.  So dereferencing the `email` thread will be a bit of a gamble.